### DISCUSSION TITLE
Outlook to Tiff file

### DISCUSSION THREAD

**Discussion by marcotam604** [ImageMagick/ImageMagick#4454] [Help]
Hi,
Does ImageMagick can convert the outlook message to a tiff fill.?

Thankyou!

**Comment by snibgo** [comment]
I don't understand the question.

**Reply by marcotam604** [reply]
Oh sorry. It's a typo. I want to know that ImageMagick can convert the outlook mail message to a tiff file format or not?

**Comment by snibgo** [comment]
ImageMagick can rasterize text to make a tiff image. The text can be inserted into a command, or a command can read the text from a file.

IM knows nothing about outlook mail messages.
